### PR TITLE
Add tubemap

### PR DESCRIPTION
**Pull request by LaurenceKuhl**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/crisprseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/crisprseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,11 +34,9 @@ On release, automated continuous integration tests run the pipeline on a full-si
...
-For crispr targeted:
-
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/nf-core/crisprseq/dev/docs/images/crisprseq_targeted_metro_map_dark.png">
-  <img alt="Text changing depending on mode. Light: 'So light!' Dark: 'So dark!'" src="https://raw.githubusercontent.com/nf-core/crisprseq/dev/docs/images/crisprseq_targeted_metro_map.png">
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/nf-core/crisprseq/dev/docs/images/crisprseq_metromap_all.png">
+  <img alt="Text changing depending on mode. Light: 'So light!' Dark: 'So dark!'" src="https://raw.githubusercontent.com/nf-core/crisprseq/dev/docs/images/crisprseq_metromap_all.png">
 </picture>
 
 1. Merge paired-end reads ([`Pear`](https://cme.h-its.org/exelixis/web/software/pear/doc.html))
```
Comment (mirpedrol) [inline README.md:42]: We could rephrase the points to merge both subworkflows. Or add teh tube-map as a common figure but keep "For crispr targeted:" from line 37

**Review by mirpedrol — COMMENTED** [review]
Looks very good! 🤩 Only a couple of details:
- the UMI clustering box is missing a magnifying glass or the gray arrow.
- The legend descriptions should say if it's targeted or screening, and change the current description specially for the purple line.
We can also delete the old tube maps :)

File: README.md (hunk 1 above)
Comment (LaurenceKuhl) [reply README.md:42]: I'm not sure i see what you mean? But i'd remove the old tubemap

Comment (mirpedrol) [reply README.md:42]: I agree with removing the tubemap :) I mean we explain the points of each subworkflow with https://github.com/nf-core/crisprseq/blob/master/README.md?plain=1#L58 and https://github.com/nf-core/crisprseq/blob/master/README.md?plain=1#L33
So I wouldn't delete the one for targeted https://github.com/nf-core/crisprseq/pull/128/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L37

**Comment by LaurenceKuhl** [comment]
hi @mirpedrol  somehow the test targeted isn't running? Is this an error you've encountered before and you just rerun it?

**Comment by mirpedrol** [comment]
> hi @mirpedrol somehow the test targeted isn't running? Is this an error you've encountered before and you just rerun it?

I haven't seen this error before, let's try rerunning it 🤞

**Review by mirpedrol — APPROVED** [review]
🤩

**Pull request merged**